docarray/typing/tensor/image/abstract_image_tensor.py

```diff
@@ -31,6 +36,22 @@ def to_bytes(self, format: str = 'PNG') -> bytes:
 
         return img_byte_arr
 
+    def save(self: 'T', file_path: str) -> None:
```

Comment:
I don't think you need the T here. Self is always from the class
```suggestion
    def save(self, file_path: str) -> None:
```